.gitlab/source_test/ebpf.yml

```diff
@@ -14,12 +14,16 @@
   - cp $CI_PROJECT_DIR/pkg/ebpf/bytecode/build/offset-guess-debug.o $CI_PROJECT_DIR/.tmp/binary-ebpf/offset-guess-debug.o
   - cp $CI_PROJECT_DIR/pkg/ebpf/bytecode/build/usm.o $CI_PROJECT_DIR/.tmp/binary-ebpf/usm.o
   - cp $CI_PROJECT_DIR/pkg/ebpf/bytecode/build/usm-debug.o $CI_PROJECT_DIR/.tmp/binary-ebpf/usm-debug.o
+  - cp $CI_PROJECT_DIR/pkg/ebpf/bytecode/build/shared-libraries.o $CI_PROJECT_DIR/.tmp/binary-ebpf/shared-libraries.o
+  - cp $CI_PROJECT_DIR/pkg/ebpf/bytecode/build/shared-libraries-debug.o $CI_PROJECT_DIR/.tmp/binary-ebpf/shared-libraries-debug.o
   - cp $CI_PROJECT_DIR/pkg/ebpf/bytecode/build/dns.o $CI_PROJECT_DIR/.tmp/binary-ebpf/dns.o
   - cp $CI_PROJECT_DIR/pkg/ebpf/bytecode/build/dns-debug.o $CI_PROJECT_DIR/.tmp/binary-ebpf/dns-debug.o
   - cp $CI_PROJECT_DIR/pkg/ebpf/bytecode/build/conntrack.o $CI_PROJECT_DIR/.tmp/binary-ebpf/conntrack.o
   - cp $CI_PROJECT_DIR/pkg/ebpf/bytecode/build/conntrack-debug.o $CI_PROJECT_DIR/.tmp/binary-ebpf/conntrack-debug.o
   - cp $CI_PROJECT_DIR/pkg/ebpf/bytecode/build/co-re/usm-debug.o $CI_PROJECT_DIR/.tmp/binary-ebpf/co-re/usm-debug.o
   - cp $CI_PROJECT_DIR/pkg/ebpf/bytecode/build/co-re/usm.o $CI_PROJECT_DIR/.tmp/binary-ebpf/co-re/usm.o
+  - cp $CI_PROJECT_DIR/pkg/ebpf/bytecode/build/co-re/shared-libraries-debug.o $CI_PROJECT_DIR/.tmp/binary-ebpf/co-re/shared-libraries-debug.o
+  - cp $CI_PROJECT_DIR/pkg/ebpf/bytecode/build/co-re/shared-libraries.o $CI_PROJECT_DIR/.tmp/binary-ebpf/co-re/shared-libraries.o
   - cp $CI_PROJECT_DIR/pkg/ebpf/bytecode/build/co-re/oom-kill.o $CI_PROJECT_DIR/.tmp/binary-ebpf/co-re/oom-kill.o
   - cp $CI_PROJECT_DIR/pkg/ebpf/bytecode/build/co-re/tcp-queue-length.o $CI_PROJECT_DIR/.tmp/binary-ebpf/co-re/tcp-queue-length.o
   - cp $CI_PROJECT_DIR/pkg/ebpf/bytecode/build/co-re/tracer-fentry.o $CI_PROJECT_DIR/.tmp/binary-ebpf/co-re/tracer-fentry.o
```

Comment:
Missing runtime `shared-libraries.c` file in this section

Reply:
Good catch 😅